h2o-docs/src/product/data-science/svm.rst

```diff
@@ -0,0 +1,69 @@
+
+-  `ignore_const_cols <algo-params/ignore_const_cols.html>`__: Specify whether to ignore constant
+   training columns, since no information can be gained from them. This
+   option is enabled by default.
+
+-  **hyper_param**: Specify the penalty parameter C of the error term. This value defaults to 1.
+
+-  **kernel_type**: Specify the type of kernel to use. Note that currently only `gaussian` is supported.
+
+-  **gamma**: Specify the coefficient of the kernel. This value defaults to -1.
+
+-  **rank_ratio**: Specify the desired rank of the `Incomplete Cholesky Facorization <https://en.wikipedia.org/wiki/Incomplete_Cholesky_factorization>`__ (ICF) matrix expressed as an ration of number of input rows.
+
+-  **positive_weight**: Specify the weight of the positive (+1) class of observations. This value defaults to 1.
+
+-  **negative_weight**: Specify the weight of the negative (-1) class of observations. This value defaults to 1.
+
+-  **sv_threshold**: Specify the threshold for accepting a candidate observation into the set of support vectors. This value defaults to 0.0001.
+
+-  **fact_threshold**: Specify the convergence threshold of the `Incomplete Cholesky Facorization <https://en.wikipedia.org/wiki/Incomplete_Cholesky_factorization>`__ (ICF)
+
+-  **feasible_threshold**: Specify the convergence threshold for primal-dual residuals in the `Interior Point Method <https://en.wikipedia.org/wiki/Interior-point_method>`__ (IPM) iteration. This value defaults to 0.001.
+
+-  **surrogate_gap_threshold**: Specify the feasibility criterion of the surrogate duality gap (eta). This value defaults to 0.001.
+
+-  **mu_factor**: Specify to increase the mean value by this factor. This value defaults to 10.
+
+-  `max_iterations <algo-params/max_iterations.html>`__: Specify the maximum number of iteration of the algorithm. This value defaults to 200.
+
+-  `seed <algo-params/seed.html>`__: Specify the random number generator (RNG) seed for
+   algorithm components dependent on randomization. The seed is
+   consistent for each H2O instance so that you can create models with
+   the same starting conditions in alternative configurations.
+
+Interpreting an SVM Model
+~~~~~~~~~~~~~~~~~~~~~~~~~
+
+SVM Algorithm
+~~~~~~~~~~~~~
+
+References
+~~~~~~~~~~
+
+Implementation of ICF based on https://static.googleusercontent.com/media/research.google.com/en//pubs/archive/34638.pdf
```

Comment:
The references section should be a citation:
E.Y. Chang, K. Zhu , H. Wang, H. Bai, J. Li, Z. Qiu, H. Cui, Parallelizing support vector machines on distributed computers, in Proceedings of NIPS, 2007 <a href="http://papers.nips.cc/paper/3202-parallelizing-support-vector-machines-on-distributed-computers.pdf">Google Scholar</a>

You can see an example of the correct citation in the References section on this page:
https://link.springer.com/chapter/10.1007/978-3-642-20429-6_10

Reply:
Thanks for the info - adjusted that section in a new commit.